File: java/src/main/java/gov/doc/ntia/sigmf/ext/global/emitter/Transmitter.java
package gov.doc.ntia.sigmf.ext.global.emitter;

import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.HashMap;
import java.util.Map;

@JsonInclude(JsonInclude.Include.NON_NULL)
public class Transmitter {

    @JsonProperty(value="model", required =  false)
    protected String model;

    protected Map<String, Object> otherFields = new HashMap<>();

    public String getModel() {
        return model;
    }

    public void setModel(String model) {
        this.model = model;
    }

    @JsonAnyGetter
    public Map<String, Object> getOtherFields(){
        return otherFields;
    }

    @JsonAnySetter
    public void add(String key, Object value){
        otherFields.put(key, value);
    }


}

File: Csharp/sigmf/sigmf/ext/global/emitter/Transmitter.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace sigmf
{
    public class Transmitter
    {
        [JsonProperty("model")]
        protected string model;

        protected Dictionary<string, object> otherfields = new Dictionary<string, object>();

        public string Model { get => model; set => model = value; }

        public Dictionary<string, object> getOtherFields()
        {
            return otherfields;
        }
        public void add(string key, object value)
        {
            otherfields.Add(key, value);
        }
    }
}